src/style.css

```diff
@@ -1 +1,86 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14.6px;
+  text-transform: uppercase;
+}
+
+ul {
+  padding: 0;
+  margin: 0;
+  padding-inline-start: 0;
+}
+
+a {
+  text-decoration: none;
+
+}
```

Comment:
check out point 1 in the [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)